frontend/server/src/Controllers/Problem.php

```diff
@@ -1256,14 +1263,15 @@ private static function validateDownload(\OmegaUp\Request $r) {
     /**
      * Validate problemset Details API
      *
-     * @param \OmegaUp\DAO\VO\Problems $problem
-     * @param $problemsetId
-     * @param $contestAlias
-     * @return Array
+     * @return null|array{contest?:\OmegaUp\DAO\VO\Contests, problemset: \OmegaUp\DAO\VO\Problemsets}
      * @throws \OmegaUp\Exceptions\ApiException
      * @throws \OmegaUp\Exceptions\NotFoundException
      */
-    private static function validateProblemset(\OmegaUp\DAO\VO\Problems $problem, $problemsetId, $contestAlias = null) {
+    private static function validateProblemset(
+        \OmegaUp\DAO\VO\Problems $problem,
+        ?int $problemsetId,
+        string $contestAlias = null
```

Comment:
```suggestion
        ?string $contestAlias = null
```

como ese parámetro tiene `null` como valor default, mejor hay que hacerlo nullable.

Reply:
Done